paper-icon-button.html

```diff
@@ -135,6 +139,23 @@
        */
       icon: {
         type: String
+      },
+
+      /**
+       * Specifies the alternate text for the button, for accessibility.
+       */
+      alt: {
+        type: String,
+        observer: "_altChanged"
+      }
+    },
+
+    _altChanged: function(newValue, oldValue) {
+      var label = this.getAttribute('aria-label');
+
+      // Don't stomp over a user-set aria-label.
+      if (!label || oldValue == label) {
```

Comment:
Won't this break if a user sets an `aria-label` which happens to be the same as `alt`, then changes `alt`?

Do we care about that case?